bucketlist/app.py

```diff
@@ -0,0 +1,84 @@
+from flask import Flask
+import config
+from bucketlist.bucketlist_api import AppAPI
+from flask_restful import Api, Resource
+from flask_jwt import JWT, jwt_required
+from bucketlist.auth import Auth
+
+app = Flask(__name__)
+app.config.from_object(config)
+api = Api(app, catch_all_404s=True)
+API = AppAPI()
+auth = Auth()
+jwt = JWT(app, auth.authenticate, auth.identity)
+
+
+class RegisterAPI(Resource):
+    """
+    RegisterAPI
+    """
+    def post(self):
+        response = API.register()
+        return response
+
+
+class BucketlistItemAPI(Resource):
+    """
+    BucketlistItemAPI
+
+    Handle all requests to /bucketlists/<int:list_id>/items/<int:item_id>
+    """
+    @jwt_required()
+    def post(self, list_id=None):
+        response = API.create_bucketlist_item(list_id)
+        return response
+
+    @jwt_required()
+    def put(self, list_id=None, item_id=None):
+        response = API.update_bucketlist_item(list_id, item_id)
+        return response
+
+    @jwt_required()
+    def delete(self, list_id=None, item_id=None):
+        response = API.delete_bucketlist_item(list_id, item_id)
+        return response
+
+
+class BucketListAPI(Resource):
```

Comment:
Move this class to a file inside the bucket list app. Reason being this is the file that runs your whole app(modularizing your classes)